### PR TITLE
[TIMOB-26131] Android: Do not copy geolocation libs

### PR DESCRIPTION
**Pull request by garymathews**
- To prevent play services conflicts, prevent copying bundled `play-service-*.jar` files

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-26131)

### CODE REVIEW

File: build/android.js [hunk 1]
```diff
@@ -111,6 +111,12 @@ Android.prototype.package = function (packager, next) {
 		function (cb) {
 			const moduleDirs = fs.readdirSync(path.join(ANDROID_ROOT, 'modules'));
 			async.each(moduleDirs, function (dir, callback) {
+
+				// skip geolocation
+				if ([ 'gelocation' ].includes(dir)) {
```
Comment (hansemannn) [inline build/android.js]: gelocation?

Comment (garymathews) [reply build/android.js]: :)

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by hansemannn** [comment]
@garymathews Jenkins seems to continuously fail with this PR. Did you test the changes locally already?

File: build/android.js [hunk 2]
```diff
@@ -111,6 +111,12 @@ Android.prototype.package = function (packager, next) {
 		function (cb) {
 			const moduleDirs = fs.readdirSync(path.join(ANDROID_ROOT, 'modules'));
 			async.each(moduleDirs, function (dir, callback) {
+
+				// skip geolocation
+				if ([ 'geolocation' ].includes(dir)) {
+					return;
```
Comment (sgtcoolguy) [inline build/android.js]: You're never calling the callback, so I don't think this ever properly returns.

**Pull request merged**